examples/experimental/dagster-airlift/dagster_airlift/core/load_defs.py

```diff
@@ -323,11 +330,12 @@ def replace_assets_in_defs(
     )
 
 
-def assets_def_of_defs(defs: Definitions) -> Iterator[AssetsDefinition]:
-    for asset in defs.assets or []:
-        asset = check.inst(  # noqa: PLW2901
-            asset,
-            (AssetSpec, AssetsDefinition),
-            "Expected passed assets to all be AssetsDefinitions or AssetSpecs.",
-        )
-        yield asset if isinstance(asset, AssetsDefinition) else external_asset_from_spec(asset)
+def enrich_airflow_mapped_assets(
+    mapped_assets: Iterable[MappedAsset],
+    airflow_instance: AirflowInstance,
+) -> Sequence[AssetsDefinition]:
```

Comment:
do we know that the outputs here are `AssetsDefinition`, since the input could be specs?

Reply:
oh I see, `_apply_airflow_data_to_specs` forcibly definition-izes the specs, makes sense

Reply:
yup